### DISCUSSION TITLE
Is AFFiNE desktop send usage data?

### DISCUSSION THREAD

**Discussion by fang2hou** [toeverything/AFFiNE#2527] [General]
Hi guys 👋

I am quite impressed with AFFiNE and I can't wait to install it on my company's MacBook to improve productivity.

However, in adherence to my company's software usage policies, I want to double check the privacy rules. Does the AFFiNE desktop application transmit usage data or device information when AFFiNE Cloud is disabled?

**Comment by HeJiachen-PM** [comment]
Application error logs maybe sent anonymously with the current version. We will provide a strict offline mode later.
